test/jdk/javax/swing/JMenuItem/RightLeftOrientation.java

```diff
@@ -54,6 +54,17 @@
  * @run main/manual RightLeftOrientation windows
  */
 
+/*
+ * @test id=aqua
+ * @bug 8371508
+ * @requires (os.family == "mac")
```

Comment:
I guess now the test can be run in window and mac OS ? Any idea on Nimbus L&F in windows os ?